incubator/prometheus-alerts/values.yaml

```diff
@@ -9,4 +9,6 @@ labels:
 alerts:
   cluster_cpu_low:
      warning: 40
-     critical: 20
\ No newline at end of file
+     critical: 20
+  pod_restarts:
+     more_then: 5
```

Comment:
I think stick with `critical` and `warning`; some sort of inequality is always implied.